qml/components/settingsPage/SettingsSession.qml

```diff
@@ -0,0 +1,248 @@
+/*
+    Copyright (C) 2020-21 Sebastian J. Wolf and other contributors
+
+    This file is part of Fernschreiber.
+
+    Fernschreiber is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Fernschreiber is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Fernschreiber. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import WerkWolf.Fernschreiber 1.0
+import "../../components"
+import "../../js/twemoji.js" as Emoji
+import "../../js/functions.js" as Functions
+
+AccordionItem {
+    text: qsTr("Sessions")
+    Component {
+        Column {
+            id: activeSessionsItem
+            bottomPadding: Theme.paddingMedium
+            property variant activeSessions;
+            property bool loaded : false;
+            readonly property var userInformation : tdLibWrapper.userInformation
+
+            Component.onCompleted: {
+                if (!activeSessions) {
+                    tdLibWrapper.getActiveSessions();
+                } else {
+                    activeSessionsItem.loaded = true;
+                }
+            }
+
+            Connections {
+                target: tdLibWrapper
+                onSessionsReceived: {
+                    activeSessionsItem.activeSessions = sessions;
+                    activeSessionsItem.loaded = true;
+                }
+                onOkReceived: {
+                    if (request === "terminateSession") {
+                        appNotification.show(qsTr("Session was terminated"));
+                        activeSessionsItem.loaded = false;
+                        tdLibWrapper.getActiveSessions();
+                    }
+                }
+            }
+
+            ResponsiveGrid {
+                bottomPadding: Theme.paddingMedium
+
+                Loader {
```

Comment:
It doesn't make sense to place a single item into a grid.

Reply:
It's two, but still you're right. A Flow then? See 185c8f7

Reply:
If it's always a single column then a Column would be appropriate. The purpose of that grid thing is to save vertical space in landscape (or on a large screen) by re-arranging items in two columns.

Reply:
Which would still make sense as there are two items in that Accordeon (current session info and list of devices). The current variant with the Flow works as you describe, but is inconsistent with how the other Accordeons do it.

I'll look at a rework with ResponsiveGrid.

Reply:
That was easy, Flow to ResponsiveGrid again. Squashed.